airgun/entities/host.py

```diff
@@ -324,6 +320,10 @@ def step(self, *args, **kwargs):
         entity_name = kwargs.get('entity_name')
         self.parent.search(entity_name)
         self.parent.table.row(name=entity_name)['Name'].widget.click()
+        host_view = NewHostDetailsView(self.parent.browser)
+        host_view.wait_displayed()
+        host_view.dropdown.wait_displayed()
+        host_view.dropdown.item_select('Legacy UI')
```

Comment:
`host_view.dropdown.fill('Legacy UI')` is more robust is not that working for you?

Reply:
@omkarkhatavkar 
I don't understand why you are asking this question?   
PF4 dropdown - `from widgetastic_patternfly4.ouia import Dropdown` inherits from `BaseDropdown` and it has `item_select`. 
Why should I even try to use the fill method, that you are proposing?

Reply:
Fair enough I guess the pattern was fill method usually used the item select options. You are correct it is not presently part of the BaseDropdown.